theme/templates/resource-landing-page/top-right-buttons.html

```diff
@@ -66,6 +66,23 @@
                         {% endif %}
                     </form>
                 {% endif %}
+                {% if user.is_authenticated and rights.statement %}
```

Comment:
May be it is better to use a new context variable ```show_resource_copy_option``` and set that in the page preprocessor to a bool value.